### PR TITLE
Comment in review

### PR DESCRIPTION
**Pull request by tiennh1995**

### CODE REVIEW

**Comment by qtsoul95** [comment]
Ok

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    @review = Review.find_by_id params[:review_id]
+    check_null @review
+    @comment.review_id = @review.id
+    if @comment.save
+      respond_to do |format|
+        format.html {redirect_to @review.book}
+        format.js
+      end
+    else
+      flash.now[:danger] = t "comments.danger"
```
Comment (yongsokheng) [inline app/controllers/comments_controller.rb]: Neu khong save duoc thi no se lam gi? Phai co render hay redirect_to khi khong save duoc

File: app/views/shared/_rated.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="ratings" id="ratings_<%= @book.id %>">
+  <p class="pull-right"><%= @book.reviews.count %>
+    <%= t "reviews.title" %>
+  </p>
+  <p>
+    <%= @rated.round(2) %>
```
Comment (yongsokheng) [inline app/views/shared/_rated.html.erb]: bo ()

File: app/views/shared/_review_update_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<%= form_for [@book, review], html: {method: :patch}, remote: true do |f| %>
```
Comment (yongsokheng) [inline app/views/shared/_review_update_form.html.erb]: thu bo html: {method: :patch} co duoc ko?

File: app/views/shared/_review_form.html.erb [hunk 4]
```diff
@@ -1,10 +1,10 @@
-<%= form_for [@book, @review] do |f| %>
+<%= form_for [@book, review] do |f| %>
   <div class="row" style="display: inline;">
```
Comment (yongsokheng) [inline app/views/shared/_review_form.html.erb]: nen viet vao trong css

**Comment by yongsokheng** [comment]
OK

**Pull request merged**